src/app/_components/PortalComponents/Portal/users/UsersTable.tsx

```
@@ -1,39 +1,40 @@
+    },
+    onSuccess: async () => {
+      await toast.promise(utils.portal.getClerkUsers.invalidate(), {
         loading: "Saving...",
         success: "User updated successfully!",
       });
-    });
+    },
+  });
+  const { user } = useUser();
+  const handleChange = (userId: string, role: UserRole) => {
+    mutateUserRole.mutate({ userId, role });
```

ESLint

[error] Expected object keys to be in ascending order. 'role' should be before 'userId'.

(sort-keys)